docs/product-user-guide/src/main/asciidoc/chap-deploying-projects.adoc

```diff
@@ -378,9 +378,9 @@ image::task-name.png[]
 +
 image::new-fl-final.png[]
 +
-After the filter with a specified restriction is applied, variables associated with the task appear in the list of selectable columns.
+After the filter with a specified restriction is applied, the set of configurable columns is based on the specific custom task filter and contains the following column:
```

Comment:
"the following columns:" ? or "the following column options:" ? Something seems off. Otherwise, PR looks good.

Reply:
Done. Thanks!